src/components/Questions/Body.vue

```diff
@@ -305,6 +295,40 @@ export default defineComponent({
         (!props.isPortrait && !isQuestionImagePresent.value),
     }));
 
+    const hasCharLimit = computed(() => props.maxCharLimit != -1);
+
+    const maxCharLimitClass = computed(() => {
+      // class for the character limit text
+      if (charactersLeft.value > 0.2 * props.maxCharLimit) {
+        return "text-gray-400";
+      } else if (charactersLeft.value > 0.1 * props.maxCharLimit) {
+        return "text-yellow-500";
+      } else return "text-red-400";
+    });
+    const charactersLeft = computed(() => {
+      // number of characters left for the subjective answer if a limit is given
+      return props.maxCharLimit - currentAnswerLength.value;
+    });
+    const currentAnswerLength = computed(() => {
+      // length of the current answer (for subjective question)
+      if (state.subjectiveAnswer == null) return 0;
+      return state.subjectiveAnswer.length;
+    });
+    const defaultAnswer = computed(() => {
+      // the default answer to be shown for the subjective question
+      if (
+        props.submittedAnswer != null &&
+        typeof props.submittedAnswer == "string"
+      ) {
+        return props.submittedAnswer;
```

Comment:
Why are you returning `submittedAnswer` when the variable name is `defaultAnswer?